client/layout/index.jsx

```diff
@@ -274,6 +276,7 @@ class Layout extends Component {
 
 		return (
 			<div className={ sectionClass }>
+				<SympathyDevWarning />
```

Comment:
One thing I'd still like to do is conditionally render this only if this is the development environment. Feel free to abstract this into the component itself, or to put it in the layout components, but I believe that's one of the last pieces we're missing here.

Reply:
Good call! I have done that here bf7d822be9545857711b1737bb09c8655eb056c6